test/unit/org/apache/cassandra/cql3/GcCompactionTest.java

```diff
@@ -264,7 +267,7 @@ public void testGarbageCollectOrder() throws Throwable
         assertEquals(CompactionManager.AllSSTableOpStatus.SUCCESSFUL, status);
 
         SSTableReader[] tables = cfs.getLiveSSTables().toArray(new SSTableReader[0]);
-        Arrays.sort(tables, (o1, o2) -> Integer.compare(o1.descriptor.generation, o2.descriptor.generation));  // by order of compaction
+        Arrays.sort(tables, Comparator.comparing(o -> o.descriptor.generation, SSTableUniqueIdentifierFactory.COMPARATOR));  // by order of compaction
```

Comment:
Nit: we can use `SSTableReader.generationComparator`:
```suggestion
        Arrays.sort(tables, SSTableReader.generationComparator);  // by order of compaction
```